File: CourseProjects/pythonRecap/start.js
let x = 10;

if (x < 10) {
  console.log('x is less than 10');
} else {
  console.log('x is greater than 10');
}

let zahl1 = 10;
var zahl2 = 20;
const zahl3 = 30;

function addieren1(a, b) {
  return a + b;
}

const addieren2 = (a, b) => a + b;

console.log(addieren1(10, 20));
console.log(addieren2(10, 20));

i = 0;
while (i < 10) {
  console.log(i);
  i++;
}

for (i = 0; i < 10; i++) {
  console.log(i);
}

let fruechte = ['Apfel', 'Birne', 'Erdbeere'];
fruechte.push('Kiwi');

File: CourseProjects/pythonRecap/start.py
x = 10

if x < 10:
    print("x is less than 10")
elif x > 10:
    print("x is greater than 10")

zahl1 = 10
zahl2 = 20
zahl3 = 30

def addieren(a, b):
    return a + b

add = lambda a, b: a + b

print(add(5, 6))

print(addieren(5, 6))

i = 0
while i < 5:
    print(i)
    i += 1

for i in range(5):
    print(i)

fruechte = ["Apfel", "Birne", "Kirsche"]
fruechte.append("Zitrone")

print(fruechte)